src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function add(num) {
```

Comment:
The name of the ``num`` parameter seems a bit generic. 
It might be better to name it, for example, ``value`` to make it easier to understand.